static/src/javascripts/projects/common/modules/ui/welcome-banner.js

```diff
@@ -1,45 +1,63 @@
 define([
+    'bean',
     'fastdom',
     'common/utils/$',
     'common/utils/storage',
     'common/utils/template',
-    'common/utils/load-css-promise'
+    'common/utils/load-css-promise',
+    'common/views/svgs'
 ], function (
+    bean,
     fastdom,
     $,
     storage,
     template,
-    loadCssPromise
+    loadCssPromise,
+    svgs
 ) {
     /**
      * Rules:
      * 1st visit replace nav bar with banner
      */
     var header = document.getElementById('header'),
-        message1 = '<div class="banner-message"><%=HTML%></div>',
+        message1 = '<div class="banner-message"><span class="line-1"><%=first%></span><span class="line-2"><%=second%></span></div>',
         data = {
-            message1: {
-                HTML: '<span class="pull-left">news website</span><span class="pull-right">of the year</span>'
+            'award-winning': {
+                'first': 'award-winning news,',
+                'second': 'sport and culture'
             }
         };
 
-    function showWelcomeMessage(messageNumber) {
+    function showWelcomeMessage(messageName) {
         loadCssPromise.then(function () {
-            createAndSetHeader(messageNumber);
+            createAndSetHeader(messageName);
         });
     }
 
-    function createAndSetHeader(messageNumber) {
-        var headerDiv = document.createElement('div'),
-            msg = template(message1, data[messageNumber]);
+    function createAndSetHeader(messageName) {
+        var headerDiv = document.createElement('button'),
```

Comment:
It used to be a `div`, hence the name, but it ended up a `button` because I needed to track when it's clicked using `data-link-name`.